amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -460,6 +460,13 @@ type StakerRewards {
   isClaimed: Boolean!
 }
 
+type MotionEvent {
+  name: String!
+  initiatorAddress: String
```

Comment:
Initiator address will always be set

Reply:
@rdig there are actually some messages that don't have an initiator/staker/agent etc. like MotionFinalized, ObjectionFullyStaked, MotionFullyStakedAfterObjection, MotionFullyStaked, MotionVotingPhase.
In these cases there is no value for initiatorAddress and also it would be not be needed.